app/src/main/java/com/infomaniak/mail/ui/LaunchActivity.kt

```diff
@@ -50,7 +50,7 @@ class LaunchActivity : AppCompatActivity() {
     }
 
     private suspend fun startApp() {
-        mainViewModel.updateUserInfo()
+        mainViewModel.updateUserInfo(this)
```

Comment:
No need to pass the context as a parameter because the `ViewModel` can already contain it